front-end/page_target/SnvByVariant_Config.json

```diff
@@ -159,8 +159,8 @@
   },
   {
     "id": "OncoKBOncogeneTSG",
-    "label": "OncoKB oncogene TSG",
+    "label": "OncoKB Oncogene/TSG",
```

Comment:
For the "/" may can cause the confusion  as "/" present the mathematical meaning throughout the configuration file. 
How about to change it to " OncoKB Oncogene | TSG " or  "OncoKB Oncogene Or TSG" ?